packages/react/src/components/Forms/Field/index.tsx

```diff
@@ -31,10 +33,21 @@ export const Field = ({
       })}
     >
       {children}
-      {(text || rightText) && (
+      {(text || rightText || textItems) && (
         <div className="c__field__footer">
-          <span className="c__field__text">{text}</span>
-          <span className="c__field__text__right">{rightText}</span>
+          {(text || rightText) && (
+            <div className="c__field__footer__top">
+              <span className="c__field__text">{text}</span>
+              <span className="c__field__text__right">{rightText}</span>
+            </div>
+          )}
+          {textItems && (
+            <ul className="c__field__footer__items">
+              {textItems.map((textItem) => (
+                <li key={textItem}>{textItem}</li>
+              ))}
+            </ul>
+          )}
         </div>
       )}
```

Comment:
This may be painful on use.
* I've a array of errors that i give to my `Input`.
* if i've only one error i don't want bullet point.
* if i've multiple errors, i (may) want bullet point.

Reply:
I get it but imo cunningham is only intended to be a design system, it's on the consumer to decide how to use the lib.